lib/board_json.rb

```diff
@@ -0,0 +1,9 @@
+require 'json'
+
+class BoardJson
```

Comment:
I haven't checked, but are there other types of Board besides Json?  If not then this name has implementation detail.

Reply:
Indeed this is just poor naming on my part. It is actually a class for encoding a board as JSON. Do you think something like the below would convey that better?

class BoardEncoder
  def self.as_json(board)